snaps/learn/tutorials/transaction-insights.md

```diff
@@ -144,49 +573,81 @@ export const onTransaction: OnTransactionHandler = async ({ transaction }) => {
 ```
 
 :::tip
-If you have previously developed a dapp, you're likely familiar with accessing the Ethereum provider using `window.ethereum`.
-In a Snap, the `window` object is not available.
-Instead, when you request the `endowment:ethereum-provider` permission, your Snap is granted access to the [`ethereum` global object](../about-snaps/apis.md#snap-requests).
+If you have previously developed a dapp, you're likely familiar with accessing the Ethereum provider 
+using `window.ethereum`. In a Snap, the `window` object is not available. Instead, when you request 
+the `endowment:ethereum-provider` permission, your Snap is granted access to the 
+[`ethereum` global object](../about-snaps/apis.md#snap-requests).
 :::
 
-### 4. Build and test the Snap
+### 5. Build and test your Snap
 
-To build and test your Snap:
+Complete the following steps to build and test your Snap:
 
-1. From the command line, run `yarn start` in the root of your project.
-   This starts two development servers: one for watching and compiling the Snap, and another for the
-   React site.
-   The Snap bundle is served from `localhost:8080`, and the site is served from `localhost:8000`.
-   You should get a message that includes:
+#### 5.1. Increment version
```

Comment:
Same as comment on the "Estimate gas" tutorial – this step might fit better under "Next steps," since it's not required for building/testing.

Reply:
agree, I didn't like it in the first pass I did, but thought it may have more purpose than I knew!